tslint.json

```diff
@@ -86,6 +86,7 @@
         ],
         "no-use-before-declare": false,
         "newline-per-chained-call": false,
-        "no-dynamic-delete": false
+        "no-dynamic-delete": false,
+        "trailing-comma": false
```

Comment:
Is there a way to move this to a test specific tslint.json?

Reply:
[Yes](https://github.com/microsoft/vscode-azurefunctions/blob/master/test/tslint.json), but I want to disable it for the whole repo. It's easier to copy/paste/rearrange lines if they all have a comma at the end